eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -427,8 +451,14 @@ function VerifyPackages($pkgRepository, $artifactLocation, $workingDirectory, $a
       exit(1)
     }
   }
+  $pkgs = Get-ChildItem -Path $artifactLocation -Include $packagePattern -Recurse -File
+  return $pkgs, $ParsePkgInfoFn
+}
 
-  $pkgs = (Get-ChildItem -Path $artifactLocation -Include $packagePattern -Recurse -File)
+# Walk across all build artifacts, check them against the appropriate repository, return a list of tags/releases
+function VerifyPackages($pkgRepository, $artifactLocation, $workingDirectory, $apiUrl, $releaseSha,  $continueOnError = $false) {
```

Comment:
Make sure you verify one of the steps like the TagRepository step that calls this function to ensure it still works as expected.

Reply:
Yes, I am testing on the test pipeline to make sure all function works well.